src/lifecycle/events.js

```diff
@@ -1,5 +1,14 @@
 import matches from '../util/matches-selector';
 
+function readonly (obj, prop, val) {
+  Object.defineProperty(obj, prop, {
+    configurable: true,
```

Comment:
Yeah, if you plan on redefining it. This is true if you reuse an event object and it passes through the same lifecycle. Since you can override it in native, we should probably try and stick as closely as possible to that.